gamestonk_terminal/forex/fx_view.py

```diff
@@ -0,0 +1,544 @@
+                time_in_force = response["orders"][i]["timeInForce"]
+                print(f"Order ID: {order_id}")
+                print(f"Instrument: {instrument}")
+                print(f"Price: {price}")
+                print(f"Units: {units}")
+                print(f"Time created: {create_time}")
+                print(f"Time in force: {time_in_force}")
+                print("-" * 30)
+            except IndexError:
+                break
+    except V20Error as e:
+        print(e)
+
+
+def get_open_positions(accountID):
+    try:
+        request = positions.OpenPositions(accountID)
+        response = client.request(request)
+        for i in range(len(response["positions"])):
+            instrument = response["positions"][i]["instrument"]
+            long_units = response["positions"][i]["long"]["units"]
+            long_pl = response["positions"][i]["long"]["pl"]
+            long_upl = response["positions"][i]["long"]["unrealizedPL"]
+            short_units = response["positions"][i]["short"]["units"]
+            short_pl = response["positions"][i]["short"]["pl"]
+            short_upl = response["positions"][i]["short"]["unrealizedPL"]
+            print(f"Instrument: {instrument}\n")
+            print(f"Long Units: {long_units}")
+            print(f"Total Long P/L: {long_pl}")
+            print(f"Long Unrealized P/L: {long_upl}\n")
+            print(f"Short Units: {short_units}")
+            print(f"Total Short P/L: {short_pl}")
+            print(f"Short Unrealized P/L: {short_upl}")
+            print("-" * 30 + "\n")
+    except V20Error as e:
+        print(e)
+
+
+def get_open_trades(accountID):
+    try:
+        request = trades.OpenTrades(accountID)
+        response = client.request(request)
+        df = pd.DataFrame.from_dict(response["trades"])
+        df = df[
+            [
+                "id",
+                "instrument",
+                "initialUnits",
+                "currentUnits",
+                "price",
+                "unrealizedPL",
+            ]
+        ]
+        df = df.rename(
+            columns={
+                "id": "ID",
+                "instrument": "Instrument",
+                "initialUnits": "Initial Units",
+                "currentUnits": "Current Units",
+                "price": "Entry Price",
+            }
+        )
+        print(df)
+    except V20Error as e:
+        print(e)
+
+
+def close_trade(accountID, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="close_trade",
+        description="close a trade",
+    )
+    parser.add_argument(
+        "-i",
+        "--id",
+        dest="orderID",
+        action="store",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-u",
+        "--units",
+        dest="units",
+        action="store",
+        type=str,
+        required=False,
+    )
+    ns_parser = parse_known_args_and_warn(parser, other_args)
```

Comment:
Here again, I would allow the user to do:
`close 3` by adding that code snipped that I mentioned before.

I'll stop commenting this forward, I'll leave it up to your judgment to add it. This is just to improve user experience.